pkgs/applications/window-managers/i3/default.nix

```diff
@@ -79,16 +144,21 @@ stdenv.mkDerivation rec {
 
   separateDebugInfo = true;
 
-  passthru.tests = { inherit (nixosTests) i3wm; };
-
+  passthru.tests = {
+    inherit (nixosTests) i3wm;
+    updateScript = nix-update-script { };
```

Comment:
Typo, you mistakenly defined updateScript within the tests attrset.

Reply:
:facepalm: